### ISSUE TITLE
[Feature Request] Is it possible to fix 1440p rendering issues?

### ISSUE THREAD

**Issue by wjoe** [rebtd7/FF13Fix#9]
First of all, thank you for the mod! It's significantly improved the framerate of the game for me, I've often had issues running Final Fantasy XIII, made more difficult by playing under Linux, but between this mod and recent Proton improvements, I'm finally able to play it reliably at high settings with good performance.

There seems to be a common issue specifically with running Final Fantasy XIII at 2560x1440 resolution, that textures/models look very jagged and blocky, making it actually look worse at this resolution than at 1080p.

This is described in a few places, [PC Gaming Wiki](https://www.pcgamingwiki.com/wiki/Final_Fantasy_XIII#2560x1440_resolution_is_broken) mentions it and suggests a possible workaround, and has an [image showing the issue](https://images.pcgamingwiki.com/0/0f/Final_Fantasy_XIII_1440p_glitch.png). Steam Forums also [has a thread on the issue.](https://steamcommunity.com/app/292120/discussions/0/620700960208394599/), the suggested solution is to use GeDoSaTo, but that's not compatible with this fix, so that's not ideal. Another suggestion is to set a virtual resolution of 1 pixel higher than 1440p, but I'm not sure how to go about doing this on Linux, so I've not been able to confirm that.

I have no idea what causes this or if it's fixable, but was wondering if there's any chance a fix for this can be implemented into FF13Fix? It's not a massive issue, running in fullscreen at 1080p works fine, but it's the last missing piece of the puzzle in getting the game running perfectly.

**Comment by rebtd7** [comment]
I've spent several hours trying to understand and fix the issue the proper way bug I wasn't able to. 
Maybe someone more experienced in D3D9 would be able to do it.

For now I've added a workaround in v1.4.4 reducing the internal res to 2559x1440, which is good enough I suppose.

https://github.com/rebtd7/FF13Fix/blob/cadb6d0eedf8f7a8ce739e0ae9b23209dcb31e72/d3d9ex/AutoFix.cpp#L140-L148

The same workaround works fine in FF13-2 too 😄
